### DISCUSSION TITLE
Need a 'Galaxy Brain' ???

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#18] [Announcements]
### Hola GitHuber :octocat:
For a `galaxy Brain` tag you need to answer at least 2 discussions.
For a `galaxy Brain 🥉 ` tag you need to answer at least 8 discussions.
For a `galaxy Brain 🥈 ` tag you need to answer at least 16 discussions.
For a `galaxy Brain 🥇 ` tag you need to answer at least 32 discussions.

### Step - 1
**Go to [Galaxy Qs](https://github.com/Sriansh-raj/GitHub-Achievements/discussions/categories/galaxy-qs) section.**

<img width="960" alt="image" src="https://github.com/Sriansh-raj/GitHub-Achievements/assets/99705533/ae3f562d-a636-42ac-b39a-ab89b7cb0221">

### Step - 2
**Give correct answers to the simple questions asked.**

<img width="935" alt="image" src="https://github.com/Sriansh-raj/GitHub-Achievements/assets/99705533/aac5e949-7181-47c3-85c1-b4191ad57a25">

**The first and correct answer will be selected and marked as `answer`.**

**Comment by Sriansh-raj** [comment]
Once you get the achievement, pls enter or update your name at [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to make a **history**. 
😄

**Reply by opticaldrive** [reply]
Will do!

**Reply by NomanGhiasy** [reply]
Will do!!

**Reply by epifano-x** [reply]
Will do!!

**Reply by synac1** [reply]
Thank you, will do!

**Reply by ThusathmiSapumanage** [reply]
Will do!

**Comment by NomanGhiasy** [comment]
@Sriansh-raj A question every day would be nice :D

**Reply by opticaldrive** [reply]
More like two per day!
:)

**Reply by Sriansh-raj** [reply]
@NomanGhiasy , @V205Arduino hey guys,
     actually I was posting questions everyday, but on 17th my cat 🐱 broke my laptop screen 😭 so was discontinued for a week, but I'll continue this now on again 😄

**Reply by opticaldrive** [reply]
Hope the repair was not too expensive.

**Reply by epifano-x** [reply]
Will do!!

**Reply by synac1** [reply]
:)

**Reply by epifano-x** [reply]
galaxy Brain tag you need to answer at least 2 discussions.
For a galaxy Brain 🥉 tag you need to answer at least 8 discussions.
For a galaxy Brain 🥈 tag you need to answer at least 16 discussions.
For a galaxy Brain 🥇 tag you need to answer at least 32 discussions.

Shouldn’t it be the other way around?
Level four needs 32 discussions.

**Comment by saurabhcoded** [comment]
Well this is the only way or any anothe rway

**Reply by Sriansh-raj** [reply]
Hola @saurabhcoded, 
actually you just need two accepted answers in a qs-discussion of any repo.

I am just providing you the platform to make you guys avail with ease.

hope you get this 😄

**Reply by epifano-x** [reply]
galaxy Brain tag you need to answer at least 2 discussions.
For a galaxy Brain 🥉 tag you need to answer at least 8 discussions.
For a galaxy Brain 🥈 tag you need to answer at least 16 discussions.
For a galaxy Brain 🥇 tag you need to answer at least 32 discussions.

Shouldn’t it be the other way around?
Level four needs 32 discussions.

**Comment by opticaldrive** [comment]
galaxy Brain tag you need to answer at least 2 discussions.
For a galaxy Brain 🥉  tag you need to answer at least 8 discussions.
For a galaxy Brain 🥈  tag you need to answer at least 16 discussions.
For a galaxy Brain 🥇  tag you need to answer at least 32 discussions.

Shouldn’t it be the other way around?
Level four needs 32 discussions.

**Reply by Sriansh-raj** [reply]
Heya @V205Arduino,
So far I know you need 32 accepted answers for the level 4 badge.
So idts there's any other way. :)

**Reply by opticaldrive** [reply]
Oh what I meant is the number goes up when you answer more.

**Reply by opticaldrive** [reply]
Like this?

1: 2 answers
2: 8 answers
3: 16 answers
4: 32 answers?

**Reply by Sriansh-raj** [reply]
Yep

**Reply by opticaldrive** [reply]
Ok!

**Reply by Sriansh-raj** [reply]
<img width="269" alt="image" src="https://github.com/Sriansh-raj/GitHub-Achievements/assets/99705533/5b45da8b-a6d7-4920-9652-51470f003a8d">

@V205Arduino see this user has got `X4` as he has answered 32 even more than that 😄

**Reply by Sriansh-raj** [reply]
Hope you got this :)

**Reply by opticaldrive** [reply]
Yep!
Thanks!

**Comment by epifano-x** [comment]
galaxy Brain tag you need to answer at least 2 discussions.
For a galaxy Brain 🥉 tag you need to answer at least 8 discussions.
For a galaxy Brain 🥈 tag you need to answer at least 16 discussions.
For a galaxy Brain 🥇 tag you need to answer at least 32 discussions.

Shouldn’t it be the other way around?
Level four needs 32 discussions.

**Reply by Sriansh-raj** [reply]
Hey @epifano-x, looks like you appeared here for the first time :D
Go to [Galaxy Qs](https://github.com/Sriansh-raj/GitHub-Achievements/discussions/categories/galaxy-qs) section and get accepted answer of two questions to get the `Galaxy Brain` badge

**Comment by anshika-guleria** [comment]
This is a pretty cool way to do it ain't gonna lie.
